### PR TITLE
Code reformat

### PR DESCRIPTION
**Pull request by floonone**
Reformatted code to follow more consistent style rules, mainly related to whitespace and line breaking.
Removed unused imports.
Fixed some typos on docstrings.

### CODE REVIEW

File: uvispace/uvirobot/speedtransform.py [hunk 1]
```diff
@@ -216,33 +216,33 @@ def nonlinear_transform(self, min_A=30, max_A=100,
...
         num2 = (speed - self._min_value) * (max_A - min_A)
         den1 = self._max_value - zero_value
         den2 = zero_value - self._min_value
-        #Operations applied to values greater than zero_value
-        speed[speed>zero_value] = (num1[speed>zero_value] / den1) + min_B
-        #Operations applied to values smaller than zero_value
-        speed[speed<zero_value] = (num2[speed<zero_value] / den2) + min_A
-        #Zero values are turned to the new scale defined zero
-        speed[speed==zero_value] = scale_zero
+        # Operations applied to values greater than zero_value
```
Comment (jlrandulfe) [inline uvispace/uvirobot/speedtransform.py]: Similar as before,"operators with higher priorities are recommended to not have whitespaces...". For making the instruction more understandable, I think that the boolean operators should not be surrounded by whitespaces

File: uvispace/uvisensor/client.py [hunk 2]
```diff
@@ -118,54 +119,54 @@ def open_connection(self, ip, port):
...
-                                     'of the data acquired'.format(amount))
+                                  'of the data acquired'.format(amount))
                 except:
                     pass
                 break
             bytes += len(received_package)
             packages.append(received_package)
         try:
             rospy.logdebug('Received {} bytes of {} ({:.2f}%)\r'.format(
-                           bytes, size, (100 * float(bytes)/size)))
```
Comment (jlrandulfe) [inline uvispace/uvisensor/client.py]: indent criteria?

File: uvispace/uvisensor/imgprocessing.py [hunk 3]
```diff
@@ -134,21 +137,20 @@ def correct_distortion(self, kx=0.035, ky=0.035, only_contours=True):
...
+        # If contours is an empty list, algorithm is not outperformed.
         if only_contours and self.contours:
             for index, cnt in enumerate(self.contours):
                 distance = cnt - center
-                #Calculate the r distance. First numerator and then denominator.
+                # Calculate the r distance. First numerator and then denominator.
                 r = (distance ** 2).sum(axis=1).astype(np.float)
                 r /= (center ** 2).sum() * 2
-                coeffs = np.array([r*ky, r*kx]).transpose() + 1
+                coeffs = np.array([r * ky, r * kx]).transpose() + 1
```
Comment (jlrandulfe) [inline uvispace/uvisensor/imgprocessing.py]: operators with different priorities

File: uvispace/uvisensor/singlecamera.py [hunk 4]
```diff
@@ -89,10 +92,10 @@ def cam_task(begin_loop, end_loop, conf_file):
     begin_event = threading.Event()
     end_event = threading.Event()
     threads = [
-           threading.Thread(target=input_task, args=(begin_event, end_event)),
```
Comment (jlrandulfe) [inline uvispace/uvisensor/singlecamera.py]: indent criteria¿?

File: uvispace/uvisensor/videosensor.py [hunk 5]
```diff
@@ -196,25 +200,25 @@ def load_configuration(self, write2fpga=True):
...
         if not self.conf.sections():
             try:
                 rospy.logerr('Missing config file: {}'.format(self.filename))
             except:
                 pass
             return
-        #Sensor color thresholds parameters
+        # Sensor color thresholds parameters
         self._params['red_thresholds'] = ast.literal_eval(
-                                self.conf.get('Sensor', 'red_thresholds'))
```
Comment (jlrandulfe) [inline uvispace/uvisensor/videosensor.py]: indent criteria?

File: uvispace/uvisensor/videosensor.py [hunk 6]
```diff
@@ -226,42 +230,43 @@ def load_configuration(self, write2fpga=True):
...
+        # Write to the FPGA registers the loaded configuration.
+        # SENSOR COLOR THRESHOLDS
+        # They must be of string type.
         self.set_register('RED_THRESHOLD', self._params['red_thresholds'])
         self.set_register('GREEN_THRESHOLD', self._params['green_thresholds'])
         self.set_register('BLUE_THRESHOLD', self._params['blue_thresholds'])
-        ####CAMERA GEOMETRY PARAMETERS####
-        #They must be string type.
-        self.set_register('IMAGE_SHAPE', (self._params['width'], 
+
```
Comment (jlrandulfe) [inline uvispace/uvisensor/videosensor.py]: blank line inside the same function?

File: uvispace/uvisensor/multiplecamera.py [hunk 7]
```diff
@@ -80,98 +80,98 @@ class CameraThread(threading.Thread):
...
                     videosensor.set_tracker(self.camera, self.image)
                 continue
-            #Scale the contours obtained according to the FPGA to image ratio.
+            # Scale the contours obtained according to the FPGA to image ratio.
             contours = np.array(locations) / self.camera._scale
-            #Convert from Cartesian to Image coordinates
-            tmp = np.copy(contours[:,0])
-            contours[:,0] = contours[:,1]
-            contours[:,1] = tmp
+            # Convert from Cartesian to Image coordinates
```
Comment (jlrandulfe) [inline uvispace/uvisensor/multiplecamera.py]: Incoherent with NumPy style

File: uvispace/uvisensor/multiplecamera.py [hunk 8]
```diff
@@ -236,110 +237,109 @@ def __init__(self, triangles, ntriangles, conditions, inborders,
...
                 #
                 if self._triangles[index]:
-                    #Check that the element is not of None type.
+                    # Check that the element is not of None type.
                     if self._triangles[index]['1']:
                         triangle = self._triangles[index]['1']
-                        #Evaluate if triangle is in borders region.
+                        # Evaluate if triangle is in borders region.
                         self._inborders[index]['1'] = triangle.in_borders(
-                                                    self.quadrant_limits[index])
```
Comment (jlrandulfe) [inline uvispace/uvisensor/multiplecamera.py]: indent criteria

File: uvispace/uvisensor/multiplecamera.py [hunk 9]
```diff
@@ -236,110 +237,109 @@ def __init__(self, triangles, ntriangles, conditions, inborders,
...
+                # Check in the other quadrants if the triangle is in borders.
                 #
                 for index2, quadrant in enumerate(self.quadrant_limits):
-                    #Do not run the function for the current quadrant.
+                    # Do not run the function for the current quadrant.
                     if index2 == index:
                         continue
                     self._inborders[index2]['1'] = triangle.in_borders(
-                                        self.quadrant_limits[index2])
-                    #Update triangles[index2] if there is not any tracker
```
Comment (jlrandulfe) [inline uvispace/uvisensor/multiplecamera.py]: indent criteria

File: uvispace/uvisensor/multiplecamera.py [hunk 10]
```diff
@@ -236,110 +237,109 @@ def __init__(self, triangles, ntriangles, conditions, inborders,
...
-                    #initialized and UGV is within borders of the Camera.
-                    if (self._inborders[index2]['1'] and not 
-                                            self._triangles[index2]):
```
Comment (jlrandulfe) [inline uvispace/uvisensor/multiplecamera.py]: indent criteria

File: uvispace/uvisensor/multiplecamera.py [hunk 11]
```diff
@@ -236,110 +237,109 @@ def __init__(self, triangles, ntriangles, conditions, inborders,
...
+                        self.quadrant_limits[index2])
+                    # Update triangles[index2] if there is not any tracker
+                    # initialized and UGV is within borders of the Camera.
+                    if (self._inborders[index2]['1'] and not
+                    self._triangles[index2]):
                         self._ntriangles[index2]['1'] = copy.copy(triangle)
                         self._reset_flags[index2]['1'] = False
                         rospy.loginfo("New triangle in Camera{}".format(
-                                                index2))
```
Comment (jlrandulfe) [inline uvispace/uvisensor/multiplecamera.py]: indent

File: uvispace/uvisensor/geometry.py [hunk 12]
```diff
@@ -79,45 +81,45 @@ def local2global(self, offsets, K=None, image2cartesian=True):
...
+        # Assess K and assign value to 'scale' attribute if it is valid.
         if K is None:
             K = self._scale
         elif K <= 0:
             raise ValueError("The scale ratio K must be greater than 0")
         else:
             self._scale = K
-        self.vertices[:,0] = offsets[0] - self.vertices[:,0]
-        self.vertices[:,1] -= offsets[1]
+        self.vertices[:, 0] = offsets[0] - self.vertices[:, 0]
```
Comment (jlrandulfe) [inline uvispace/uvisensor/geometry.py]: Incoherent with NumPy style

File: uvispace/uvisensor/geometry.py [hunk 13]
```diff
@@ -79,45 +81,45 @@ def local2global(self, offsets, K=None, image2cartesian=True):
...
         if image2cartesian:
-            #Convert the vertices coordinates
-            tmp = np.copy(self.vertices[:,0])
-            self.vertices[:,0] = self.vertices[:,1]
-            self.vertices[:,1] = tmp
-            #Convert the barycenter coordinates
+            # Convert the vertices coordinates
+            tmp = np.copy(self.vertices[:, 0])
+            self.vertices[:, 0] = self.vertices[:, 1]
+            self.vertices[:, 1] = tmp
```
Comment (jlrandulfe) [inline uvispace/uvisensor/geometry.py]: Incoherent with NumPy style

File: uvispace/uvisensor/geometry.py [hunk 14]
```diff
@@ -147,23 +149,23 @@ def global2local(self, offsets, K=None, cartesian2image=True):
...
         if cartesian2image:
-            #Convert the vertices coordinates
-            tmp = np.copy(self.vertices[:,0])
-            self.vertices[:,0] = self.vertices[:,1]
-            self.vertices[:,1] = tmp
-            #Convert the barycenter coordinates
+            # Convert the vertices coordinates
+            tmp = np.copy(self.vertices[:, 0])
+            self.vertices[:, 0] = self.vertices[:, 1]
+            self.vertices[:, 1] = tmp
```
Comment (jlrandulfe) [inline uvispace/uvisensor/geometry.py]: Incoherent with NumPy style

File: uvispace/uvisensor/geometry.py [hunk 15]
```diff
@@ -172,13 +174,13 @@ def global2local(self, offsets, K=None, cartesian2image=True):
                 pass
             self.cartesian = False
         self.vertices /= self._scale
-        self.vertices[:,0] = offsets[0] - self.vertices[:,0]
-        self.vertices[:,1] += offsets[1]
+        self.vertices[:, 0] = offsets[0] - self.vertices[:, 0]
+        self.vertices[:, 1] += offsets[1]
```
Comment (jlrandulfe) [inline uvispace/uvisensor/geometry.py]: Incoherent with NumPy style

File: uvispace/uvisensor/geometry.py [hunk 16]
```diff
@@ -212,21 +214,21 @@ def get_pose(self):
...
-        #The base midpoint is calculated with the other 2.
+        # If 2 sides are equal, the common vertex is the front one and
+        # The base midpoint is calculated with the other 2.
         self.base_index = np.argmin(self.sides)
-        self.midpoint = (vertices[self.base_index-1]
-                         + vertices[self.base_index-2]) / 2
-        #Calculus of the x and y distance between the midpoint and the vertex
-        #opposite to the base side.
+        self.midpoint = (vertices[self.base_index - 1]
+                         + vertices[self.base_index - 2]) / 2
```
Comment (jlrandulfe) [inline uvispace/uvisensor/geometry.py]: PEP8, operators with higher priority without spaces

File: uvispace/uvisensor/geometry.py [hunk 17]
```diff
@@ -347,34 +349,35 @@ def in_borders(self, limits, tolerance=150):
         :rtype: bool
         """
         for index in range(len(limits)):
-            #Define a segment with 2 limit points of the quadrant.
-            seg = Segment(limits[index], limits[index-1])
-            #Evaluate each vertex of the triangle.
+            # Define a segment with 2 limit points of the quadrant.
+            seg = Segment(limits[index], limits[index - 1])
```
Comment (jlrandulfe) [inline uvispace/uvisensor/geometry.py]: Incoherent with NumPy style

File: uvispace/uvirobot/plotter.py [hunk 18]
```diff
@@ -107,33 +109,33 @@ def times_plot(commtimes, waittimes):
...
     plt.savefig('{}/tmp/times_plot.eps'.format(script_path),
-                                         bbox_inches='tight')
+                bbox_inches='tight')
     plt.show()
 
 
-if __name__ == "__main__":
+def main():
     test_path = np.array(
-          [[ 0.0,  0.0],
```
Comment (floonone) [inline uvispace/uvirobot/plotter.py]: Double indentation for continuation lines, as proposed in [PEP 8](https://www.python.org/dev/peps/pep-0008/#indentation).

File: uvispace/uvisensor/videosensor.py (hunk 5 above)
Comment (floonone) [inline uvispace/uvisensor/videosensor.py]: Double indentation for continuation lines, as proposed in [PEP 8](https://www.python.org/dev/peps/pep-0008/#indentation).

File: uvispace/uvisensor/geometry.py (hunk 17 above)
Comment (floonone) [inline uvispace/uvisensor/geometry.py]: Done, fixed in 5b14cc192254f37e87c372ebfaf4b1b98d16d0cd.

File: uvispace/uvirobot/plotter.py [hunk 19]
```diff
@@ -107,33 +109,33 @@ def times_plot(commtimes, waittimes):
     legend = plt.legend(shadow=True)
     script_path = os.path.dirname(os.path.realpath(__file__))
 
-#    import pdb; pdb.set_trace()
+    #    import pdb; pdb.set_trace()
```
Comment (floonone) [inline uvispace/uvirobot/plotter.py]: Done, fixed in 5b14cc192254f37e87c372ebfaf4b1b98d16d0cd.

File: uvispace/uvirobot/serialcomm.py [hunk 20]
```diff
@@ -56,7 +58,7 @@ def __init__(self, port,
         if self._isOpen:
             self.flushInput()
 
-    #-------------------MASTER-SLAVE COMMANDS-------------------#
+    # -------------------MASTER-SLAVE COMMANDS-------------------#
```
Comment (floonone) [inline uvispace/uvirobot/serialcomm.py]: Done, fixed in 5b14cc192254f37e87c372ebfaf4b1b98d16d0cd.

File: uvispace/uvirobot/serialcomm.py [hunk 21]
```diff
@@ -98,55 +100,54 @@ def move(self, setpoint=[0,0]):
...
             fun_code = result[1]
-            if fun_code :
-                return True	   
+            if fun_code:
+                return True
             else:
-                return False 
+                return False
+
+                # -------------MASTER-SLAVE COMMANDS AUXILIAR FUNCTIONS-------------#
```
Comment (floonone) [inline uvispace/uvirobot/serialcomm.py]: Done, fixed in 5b14cc192254f37e87c372ebfaf4b1b98d16d0cd.

File: uvispace/uvirobot/serialcomm.py [hunk 22]
```diff
@@ -169,50 +170,54 @@ def read_message(self):
...
+        elif id_dest != self.MASTER_ID:
             print 'Message for other device'
- 
-        return (Rx_OK, fun_code, length, data) 
 
-    #---------- CLASS CONSTANTS ----------#
-	#message fields
+        return (Rx_OK, fun_code, length, data)
+
+        # ---------- CLASS CONSTANTS ----------#
```
Comment (floonone) [inline uvispace/uvirobot/serialcomm.py]: Done, fixed in 5b14cc192254f37e87c372ebfaf4b1b98d16d0cd.

File: uvispace/uvirobot/speedtransform.py [hunk 23]
```diff
@@ -166,13 +166,13 @@ def linear_transform(self, new_min, new_max):
             raise ValueError("Not a valid scale type: {}".format(self._scale))
         num = (self._speed - self._min_value) * (new_max - new_min)
         den = self._max_value - self._min_value
-        new_value = (num/den) + new_max
+        new_value = (num / den) + new_max
```
Comment (floonone) [inline uvispace/uvirobot/speedtransform.py]: Done, fixed in 5b14cc192254f37e87c372ebfaf4b1b98d16d0cd.

File: uvispace/uvirobot/speedtransform.py (hunk 1 above)
Comment (floonone) [inline uvispace/uvirobot/speedtransform.py]: Done, fixed in 5b14cc192254f37e87c372ebfaf4b1b98d16d0cd.

File: uvispace/uvirobot/speedtransform.py [hunk 24]
```diff
@@ -216,33 +216,33 @@ def nonlinear_transform(self, min_A=30, max_A=100,
...
         try:
-             condition = (float(min_A) < float(max_A) < float(scale_zero)
-                                            < float(min_B) < float(max_B))
+            condition = (float(min_A) < float(max_A) < float(scale_zero)
+                         < float(min_B) < float(max_B))
         except (ValueError, TypeError) as e:
             raise e
         if not (float(min_A) < float(max_A) < float(scale_zero)
-                                  < float(min_B) < float(max_B)):
+                    < float(min_B) < float(max_B)):
```
Comment (floonone) [inline uvispace/uvirobot/speedtransform.py]: Double indentation for continuation lines, as proposed in [PEP 8](https://www.python.org/dev/peps/pep-0008/#indentation).

File: uvispace/uvisensor/multiplecamera.py (hunk 8 above)
Comment (floonone) [inline uvispace/uvisensor/multiplecamera.py]: Double indentation for continuation lines, as proposed in [PEP 8](https://www.python.org/dev/peps/pep-0008/#indentation).

File: uvispace/uvisensor/multiplecamera.py (hunk 9 above)
Comment (floonone) [inline uvispace/uvisensor/multiplecamera.py]: Double indentation for continuation lines, as proposed in [PEP 8](https://www.python.org/dev/peps/pep-0008/#indentation).

File: uvispace/uvisensor/multiplecamera.py (hunk 10 above)
Comment (floonone) [inline uvispace/uvisensor/multiplecamera.py]: Double indentation for continuation lines, as proposed in [PEP 8](https://www.python.org/dev/peps/pep-0008/#indentation).

File: uvispace/uvisensor/multiplecamera.py (hunk 11 above)
Comment (floonone) [inline uvispace/uvisensor/multiplecamera.py]: Double indentation for continuation lines, as proposed in [PEP 8](https://www.python.org/dev/peps/pep-0008/#indentation).

File: uvispace/uvisensor/geometry.py (hunk 12 above)
Comment (floonone) [inline uvispace/uvisensor/geometry.py]: Done, fixed in 5b14cc192254f37e87c372ebfaf4b1b98d16d0cd.

File: uvispace/uvisensor/geometry.py (hunk 13 above)
Comment (floonone) [inline uvispace/uvisensor/geometry.py]: Done, fixed in 5b14cc192254f37e87c372ebfaf4b1b98d16d0cd.

File: uvispace/uvisensor/geometry.py (hunk 14 above)
Comment (floonone) [inline uvispace/uvisensor/geometry.py]: Done, fixed in 5b14cc192254f37e87c372ebfaf4b1b98d16d0cd.

File: uvispace/uvisensor/geometry.py (hunk 15 above)
Comment (floonone) [inline uvispace/uvisensor/geometry.py]: Done, fixed in 5b14cc192254f37e87c372ebfaf4b1b98d16d0cd.

File: uvispace/uvisensor/geometry.py (hunk 16 above)
Comment (floonone) [inline uvispace/uvisensor/geometry.py]: Done, fixed in 5b14cc192254f37e87c372ebfaf4b1b98d16d0cd.

File: uvispace/uvisensor/imgprocessing.py (hunk 3 above)
Comment (floonone) [inline uvispace/uvisensor/imgprocessing.py]: Done, fixed in 5b14cc192254f37e87c372ebfaf4b1b98d16d0cd.

File: uvispace/uvisensor/multiplecamera.py (hunk 7 above)
Comment (floonone) [inline uvispace/uvisensor/multiplecamera.py]: Done, fixed in 5b14cc192254f37e87c372ebfaf4b1b98d16d0cd.

File: uvispace/uvisensor/videosensor.py (hunk 6 above)
Comment (floonone) [inline uvispace/uvisensor/videosensor.py]: Used to separate logical sections of code, as proposed in [PEP 8](https://www.python.org/dev/peps/pep-0008/#blank-lines).

**Pull request merged**